app/Console/Commands/RebuildCardsTable.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Http;
+use JsonMachine\Items;
+use App\Models\Card;
+
+class RebuildCardsTable extends Command
+{
+    protected $signature = 'scryfall:rebuild-cards';
+    protected $description = 'Rebuild the cards table daily using Scryfall bulk data';
+    
+    public function handle()
+    {
+        ini_set('memory_limit', '1G');
+        $this->info('Fetching Scryfall bulk data URL...');
+        $bulkDataUrl = 'https://api.scryfall.com/bulk-data/default-cards';
```

Comment:
I don't feel strongly about this but shouldn't we use oracle cards end point for the MVP? We won't be doing anything with the extra card art data

Reply:
I thought that specific printing and finish is relevant to card pool selection and art retrieval?
I understand oracle ids are general to the card name and text not to the printing. 
That's why I focused on scryfall and anchoring in to the scryfallid as the primary key for the card reference.

For context, when I reviewed WotC it appears they have mostly replaced oracle ids with multiverse ids for reference, with multiverse ids being the WotC print specific id.

Link refs:
https://mtgjson.com/data-models/identifiers/#scryfallid
https://mtgjson.com/data-models/identifiers/#multiverseId